__tests__/components/editor/ImportFileZone.test.js

```diff
@@ -19,7 +19,10 @@ describe('<ImportFileZone />', () => {
   describe('schema valid', () => {
     describe('schema url in JSON', () => {
       const wrapper = shallow(<ImportFileZone />)
-      let schemaUrl; let
+
+      let schemaUrl
+
+      let
```

Comment:
You might want to rebase off of the PR @mjgiarlo submitted (that should get merged first). it fixes this weirdness of the `let`s.